code/harness/FsSpecific.cpp

```diff
@@ -18,7 +18,7 @@ constexpr char kExt4MkfsOpts[] =
 // for btrfs check is not clear on whether it will try to cleanup the file
 // system some without it. It also says to be careful about using the repair
 // flag.
-constexpr char kBtrfsFsckCommand[] = "btrfs check --repair ";
+constexpr char kBtrfsFsckCommand[] = "btrfs check ";
```

Comment:
can we make this like `yes | btrfs check ` or something? Otherwise I think it won't try to fix the file system and we may run into issues later in the test suite